recipe/meta.yaml

```diff
@@ -1,5 +1,5 @@
 {% set name = "geant4" %}
-{% set version = "10.5.0" %}
+{% set version = "10.5.1" %}
 # GEANT4 data packages, versions are taken from:
 # https://github.com/Geant4/geant4/blob/v{{ version }}/cmake/Modules/Geant4DatasetDefinitions.cmake
```

Comment:
I've checked [here](https://github.com/Geant4/geant4/blob/v10.5.1/cmake/Modules/Geant4DatasetDefinitions.cmake) and the dataset versions haven't changed.

Reply:
👍